frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -28,7 +28,11 @@ abstract class FlutterRustBridgeExampleSingleBlockTest {
   /// Newlines are preserved.
   ///
   Future<int> primitiveTypes(
-      {required int myI32, required int myI64, required double myF64, required bool myBool, dynamic hint});
+      {required int myI32,
```

Comment:
Btw you need to format the code :) I will review again after code is formatted, since currently there are so many diff (and most are from code formats)

Reply:
I use the dart plugin for my vscode and it does the formatting after saving. I don't know what formatting tool you are using and do you have documentation on the formatting?

Reply:
https://github.com/fzyzcjy/flutter_rust_bridge/blob/master/justfile IIRC the `refresh_all` works